doc/microsite/src/main/tut/distage/distage-framework-docker.md

```diff
@@ -19,3 +20,581 @@ libraryDependencies += "io.7mind.izumi" %% "distage-framework-docker" % "$izumi.
+
+val distageFrameworkModules = new ModuleDef {
+  // required for docker
+  include(new DockerSupportModule[IO])
+
+  // standard distage framework modules
+  include(AppConfigModule(ConfigFactory.defaultApplication))
+  include(new CatsDIEffectModule {})
+  include(new LogIOModule[IO](StaticLogRouter.instance, false))
+}
+```
+
+The required framework modules plus `PostgresDockerModule` is sufficient to depend on Docker
+containers:
+
+```scala mdoc:to-string
+def minimalExample = {
+  val applicationModules = new ModuleDef {
+    include(new PostgresDockerModule[IO])
+    include(distageFrameworkModules)
+  }
+
+  Injector().produceGetF[IO, PostgresDocker.Container](applicationModules).use { container =>
+    val port = container.availablePorts.first(PostgresDocker.primaryPort)
+    IO(println(s"postgres is available on port ${port}"))
+  }
+}
+
+minimalExample.unsafeRunSync()
+```
+
+If the `DockerSupportModule` is not included in an application then a get of a Docker container
+dependent resource will fail with a `izumi.distage.model.exceptions.ProvisioningException`.
+
+### Config API
+
+The @scaladoc[`DockerProviderExtensions`](izumi.distage.docker.DockerContainer$$DockerProviderExtensions)
+provides additional APIs for modiying the container definition.
+
+#### `modifyConfig`
+
+Use
+@scaladoc[`modifyConfig`](izumi.distage.docker.DockerContainer$$DockerProviderExtensions#modifyConfig)
+to modify the configuration of a container. The modifier is instantiated to a `ProviderMagnet`, which
+will summon any additional dependencies.
+
+For example, to change the user of the PostgreSQL container:
+
+```scala mdoc:silent
+class PostgresRunAsAdminModule[F[_]: TagK] extends ModuleDef {
+  make[PostgresDocker.Container].fromResource {
+    PostgresDocker.make[F].modifyConfig { () => (old: PostgresDocker.Config) =>
+      old.copy(user = Option("admin"))
+    }
+  }
+}
+```
+
+Suppose `HostPostgresData` is a component provided by the application modules. This path can be
+added to the PostgreSQL container's mounts by adding to the additional dependencies of the provider
+magnet:
+
+```scala mdoc:silent
+case class HostPostgresData(path: String)
+
+class PostgresWithMountsDockerModule[F[_]: TagK] extends ModuleDef {
+  make[PostgresDocker.Container].fromResource {
+    PostgresDocker.make[F].modifyConfig {
+      (hostPostgresData: HostPostgresData) =>
+        (old: PostgresDocker.Config) =>
+          val dataMount = Docker.Mount(hostPostgresData.path, "/var/lib/postgresql/data")
+          old.copy(mounts = old.mounts :+ dataMount)
+    }
+  }
+}
+```
+
+#### `dependOnDocker`
+
+@scaladoc[`dependOnDocker`](izumi.distage.docker.DockerContainer$$DockerProviderExtensions#dependOnDocker)
+adds a dependency on a given Docker container. `distage` ensures the requested container is available
+before the dependent is provided.
+
+Suppose the system under test is a sync from PostgreSQL to Elasticsearch. One option is to use
```

Comment:
Sync? Maybe `sink`? Somewhat obtuse wording, causes a pause for me, probably needs a rewording with simpler words.